creator-node/src/routes/users.js

```diff
@@ -170,43 +170,48 @@ module.exports = function (app) {
 
     const response = {}
```

Comment:
technically this should be a let. const is supposed to be a constant value that doesn't change. i know you didn't add it, but if you could fix it that would be great!

Reply:
Yup makes sense, thanks!